.github/workflows/positron-merge-to-main.yml

```diff
@@ -113,3 +113,13 @@ jobs:
         with:
           name: run-artifacts
           path: .build/logs/smoke-tests-electron/
+
+      - name: slack-smoke-test-report
+        if: failure()
+        uses: testlabauto/xunit-slack-reporter@v2.0.1
+        env:
+          SLACK_CHANNEL: C04G3KL70TX
```

Comment:
Should the channels be the same for both? Should we add a comment which channel we're intending to send notifications to?

Reply:
Yeah, I was thinking that for both a PR test run failure and a merge to main failure we would report to #positron-builds, but I am open to suggestions.  

Added a commented noting that the channel is #positron-builds.

Reply:
switched to #positron-test-results